pom.xml

```diff
@@ -140,8 +141,115 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
why did you add Lombok?

Reply:
@alexanderkiel  Its necessary for the some of the ccdl classes